modules/project-docs/pages/sdk-full-installation.adoc

```diff
@@ -63,51 +78,57 @@ Then follow the <<post-installation,post-installation steps>>.
-
-Depending on your installation details, you may also have to add the `extension_dir` to your `PATH` variable.
-
-The packages do not include OpenSSL binaries, 
-therefore they have to be installed in the `PATH` (see https://wiki.openssl.org/index.php/Binaries) for your Windows PHP distribution.
-
-[source,console]
-----
-$ copy /b libcouchbase.dll C:\php\ext
-$ copy /b php_couchbase.dll C:\php\ext
-----
-
-|===
-|Platform|
-|Source          |https://packages.couchbase.com/clients/php/couchbase-3.2.2.tgz[couchbase-3.2.2.tgz]
-|Checksum        |https://packages.couchbase.com/clients/php/couchbase-3.2.2.sha256sum[couchbase-3.2.2.sha256sum]
-|PHP 7.4 VC15 NTS|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-7.4-nts-x64.zip[php_couchbase-3.2.2-7.4-nts-x64.zip]
-|PHP 7.4 VC15 ZTS|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-7.4-zts-x64.zip[php_couchbase-3.2.2-7.4-zts-x64.zip]
-|PHP 8.0 VC16 NTS|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.0-nts-x64.zip[php_couchbase-3.2.2-8.0-nts-x64.zip]
-|PHP 8.0 VC16 ZTS|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.0-zts-x64.zip[php_couchbase-3.2.2-8.0-zts-x64.zip]
-|PHP 8.1 VC16 NTS|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.1-nts-x64.zip[php_couchbase-3.2.2-8.1-nts-x64.zip]
-|PHP 8.1 VC16 ZTS|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.1-zts-x64.zip[php_couchbase-3.2.2-8.1-zts-x64.zip]
-|PHP 7.4 VC15 NTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-7.4-nts-x64-openssl.zip[php_couchbase-3.2.2-7.4-nts-x64-openssl.zip]
-|PHP 7.4 VC15 ZTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-7.4-zts-x64-openssl.zip[php_couchbase-3.2.2-7.4-zts-x64-openssl.zip]
-|PHP 8.0 VC16 NTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.0-nts-x64-openssl.zip[php_couchbase-3.2.2-8.0-nts-x64-openssl.zip]
-|PHP 8.0 VC16 ZTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.0-zts-x64-openssl.zip[php_couchbase-3.2.2-8.0-zts-x64-openssl.zip]
-|PHP 8.1 VC16 NTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.1-nts-x64-openssl.zip[php_couchbase-3.2.2-8.1-nts-x64-openssl.zip]
-|PHP 8.1 VC16 ZTS (TLS)|https://packages.couchbase.com/clients/php/php_couchbase-3.2.2-8.1-zts-x64-openssl.zip[php_couchbase-3.2.2-8.1-zts-x64-openssl.zip]
-|===
-
-Then follow the post-installation steps.
+Currently there are no pre-built binaries for the Windows platform of PHP SDK 4.0.0.
+Instead, you will need to follow https://github.com/couchbase/couchbase-php-client/blob/4.0.0/WINDOWS.md[these steps] to build an extension along with the PHP interpreter.
+This process will be simplified in a future 4.0.x release where the pre-compiled binaries will be published.
+
+// TODO: Add this back when this is available in a future 4.0.x release
```

Comment:
Seeing as we mention this in the release notes, I added this here to ensure users are aware.
I guess we will update this in a future SDK release @avsej?

Reply:
Yes, this is correct